_docs/manage/node.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: guide
+title: StorageOS Docs - Node diagnostics
+anchor: manage
+module: manage/node
+---
+
+# Node
+
+Node shows a per node breakdown of your storage cluster.
+
+## Overview
+
+## Listing node information
+
+To view the state of your cluster simply run:
+```bash
+storageos node ls
+
+NAME                  ADDRESS             HEALTH              SCHEDULER           VOLUMES             TOTAL               USED                VERSION             LABELS
+vol-test-2gb-lon101   46.101.50.155       Healthy 2 days      true                M: 0, R: 2          77.43GiB            5.66%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon102   46.101.50.231       Healthy 2 days      false               M: 1, R: 0          38.71GiB            5.90%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon103   46.101.51.16        Healthy 2 days      false               M: 1, R: 1          77.43GiB            5.61%               0.7 (00ab7b3 rev)
+```
+
+From the output of the command we observe that we have a 3 node StorageOS cluster and the following information:
+- IP addresses of the controller nodes
+- The health of the cluster and uptime
+- Whether the node is a scheduler node or not
```

Comment:
This is why it needs to be explained. It's not obvious to a user what a scheduler does.